libs/sdk-ui-filters/api/sdk-ui-filters.api.md

```diff
@@ -81,7 +81,7 @@ export const DateFilterHelpers: {
     defaultDateFilterOptions: IDateFilterOptionsByType;
     canExcludeCurrentPeriod: (dateFilterOption: DateFilterOption) => boolean;
     mapOptionToAfm: (value: DateFilterOption, dateDataSet: ObjRef, excludeCurrentPeriod: boolean) => IDateFilter;
-    formatAbsoluteDateRange: (from: string | Date, to: string | Date, dateFormat: string) => string;
+    formatAbsoluteDateRange: (from: string | Date, to: string | Date, dateFormat: string, isTimeEnabled: boolean) => string;
```

Comment:
This is a breaking change - `DateFilterHelpers .formatAbsoluteDateRange` is public SDK API.

Reply:
We can change it to `isTimeEnabled?: boolean`, that could fix it right?